index.js

```diff
@@ -44,8 +44,7 @@ queue.process('email', 30, require('./workers/email'));
 queue.process('sendBatchEmails', require('./workers/sendBatchEmails')(queue, db, baseUrl));
 queue.process('sendWeeklyRecapEmails', require('./workers/sendWeeklyRecapEmails')(queue, db, baseUrl));
 queue.process('amazonPayments', require('./workers/amazonPayments')(queue, db));
-
-queue.promote();
```

Comment:
Ah, jeah. I originally had my work based on @TheHollidayInn branch, which includes kue 0.11. should I put it back in or add the package.json with the updated dependencies?